src/app/components/start-load-test-widget/start-load-test-widget.component.html

```diff
@@ -0,0 +1,104 @@
+<div class="container">
+  <div class="child">
+    <button
+      *ngIf="!running"
+      class="btn btn-primary"
+      type="button"
+      (click)="startTest(); click = true"
+    >
+      Start Load Test
+    </button>
+    <button
+      class="btn btn-primary btn-sm"
+      type="button"
+      (click)="showAdvance()"
+      id="advance"
+      [disabled]="click"
+    >
+      Advance Options
+    </button>
+    <button class="btn btn-primary" (click)="stop()">Cancel</button>
+    <p *ngIf="running">Test has been running for : {{ time }} seconds</p>
+    <mat-progress-bar *ngIf="running" mode="buffer"></mat-progress-bar>
+  </div>
+  <div class="child" *ngIf="advance">
+    <form [formGroup]="AdvanceForm">
+      <ul>
+        <li>Definition of the custom settings here</li>
+        <li>Custom name to save test plans?</li>
+        <li>Example Text here!!</li>
+      </ul>
+
+      <mat-form-field>
+        <mat-label>Test plan name </mat-label>
+        <input matInput type="text" id="planName" value="" formControlName="planName" /><br />
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Number of Loops </mat-label>
+        <input matInput type="number" id="loops" min="0" max="200" formControlName="loops" /><br />
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Duration (in Seconds) </mat-label>
+        <input
+          matInput
+          type="number"
+          id="duration"
+          min="0"
+          max="200"
+          formControlName="duration"
+        /><br />
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Number of threads </mat-label>
+        <input
+          matInput
+          type="number"
+          id="thread"
+          min="0"
+          max="200"
+          formControlName="thread"
+        /><br />
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Ramp Up (in Seconds) </mat-label>
+        <input
+          matInput
+          type="number"
+          id="rampUp"
+          min="0"
+          max="200"
+          formControlName="rampUp"
+        /><br />
+      </mat-form-field>
+      <section class="example-section">
+        <mat-checkbox class="example-margin" formControlName="followRedirect"
+          >Allow Follow Redirects</mat-checkbox
+        >
+      </section>
+    </form>
+  </div>
+  <div *ngIf="running">
+    <h3>Load Test Settings</h3>
+    <p *ngIf="advance">Test Plan Name : {{ LFC.testPlanName }}</p>
+    <p>
+      Number of Loops : <strong>{{ LFC.loops }} Loops</strong>
```

Comment:
I believe that loops and duration based tests are mutually exclusive. The test will be either one or the other, not both. So this can likely be modified accordingly. Also, perhaps a table of some sort might be more appropriate.

Reply:
The backend currently is set to receive both duration and loops in the request body. They also have it so the tests will run with the duration as the priority if both parameters are received. We will revisit this to give the users the option of one or the other to avoid a confusing UI.